core/src/display_object.rs

```diff
@@ -768,6 +768,7 @@ pub fn render_base<'gc>(this: DisplayObject<'gc>, context: &mut RenderContext<'_
                 handle: handle.clone(),
                 commands: offscreen_context.commands,
                 clear: this.opaque_background().unwrap_or_default(),
+                filters: this.filters(),
```

Comment:
just to be sure: do you think us copying the filters vec (the call does a deep `.clone()` inside) is okay if dome each time here and in debug code?

Reply:
Debug code: I don't mind perf, I'd like to change `filters` to return a ref but that felt out of scope (already is vec today) which would be fine for debug

For render: It needs a clone ideally, I don't like the idea of refs leaking out to render, or things that can be interior mutable. I'd like to end up with a state that rendering can be done entirely on another thread on desktop (maybe web in a few years...) and trying to keep that design in mind, it submits work and moves on